security-admin/src/main/webapp/scripts/routers/Router.js

```diff
@@ -45,15 +45,15 @@ function(Backbone, Marionette, localization, MAppState, XAUtil){
 			"!/userprofile"		: "userProfileAction",
 			
 			"!/users/:tab"		: "userManagerAction",
-			"!/user/create"		: "userCreateAction",
-			"!/user/:id"		: "userEditAction",
+//			"!/user/create"		: "userCreateAction",
```

Comment:
Can you please add a config flag in ranger-admin-site.xml and always protect these UI elements with the config flag instead of simply commenting them out. You want to be able to only set a flag since then we'll be closer to submitting this patch to open source.